runelite-client/src/main/java/net/runelite/client/chat/ChatMessageManager.java

```diff
@@ -84,19 +90,354 @@ public void onResizableChanged(ResizeableChanged event)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentTradeChatMessage(), true),
+				ChatMessageType.TRADE);
+		}
+		if (chatColorConfig.transparentTradeChatMessageHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentTradeChatMessageHighlight(), true),
+				ChatMessageType.TRADE);
+		}
+		if (chatColorConfig.transparentServerMessage() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentServerMessage(), true),
+				ChatMessageType.SERVER);
+		}
+		if (chatColorConfig.transparentServerMessageHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentServerMessageHighlight(), true),
+				ChatMessageType.SERVER);
+		}
+		if (chatColorConfig.transparentGameMessage() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentGameMessage(), true),
+				ChatMessageType.GAME);
+		}
+		if (chatColorConfig.transparentGameMessageHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentGameMessageHighlight(), true),
+				ChatMessageType.GAME);
+		}
+		if (chatColorConfig.transparentExamine() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentExamine(), true),
+				ChatMessageType.EXAMINE_OBJECT);
+		}
+		if (chatColorConfig.transparentExamineHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentExamineHighlight(), true),
+				ChatMessageType.EXAMINE_OBJECT);
+		}
+		if (chatColorConfig.transparentExamine() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentExamine(), true),
+				ChatMessageType.EXAMINE_NPC);
+		}
+		if (chatColorConfig.transparentExamineHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentExamineHighlight(), true),
+				ChatMessageType.EXAMINE_NPC);
+		}
+		if (chatColorConfig.transparentExamine() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentExamine(), true),
+				ChatMessageType.EXAMINE_ITEM);
+		}
+		if (chatColorConfig.transparentExamineHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentExamineHighlight(), true),
+				ChatMessageType.EXAMINE_ITEM);
+		}
+		if (chatColorConfig.transparentFiltered() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.NORMAL, chatColorConfig.transparentFiltered(), true),
+				ChatMessageType.FILTERED);
+		}
+		if (chatColorConfig.transparentFilteredHighlight() != null)
+		{
+			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentFilteredHighlight(), true),
+				ChatMessageType.FILTERED);
+		}
+	}
+
 	public ChatMessageManager cacheColor(final ChatColor chatColor, final ChatMessageType... types)
```

Comment:
I would probably make this private now, so plugins will not be able to mess with it.